test/integration/test/philips_color_test.js

```diff
@@ -0,0 +1,144 @@
+'use strict';
+
+const chakram = require('chakram'), expect = chakram.expect;
+
+var Prepper = require('../lib/testPrepper.js');
+
+function translateHSB(h,s,b) {
```

Comment:
Oh, because i thought only this test would use it - technically, philips server doesn't do translation - the foxbox does, to the formate that philips server accepts